src/msw/listctrl.cpp

```diff
@@ -3471,16 +3476,16 @@ void wxListCtrl::OnPaint(wxPaintEvent& event)
     const bool drawVRules = HasFlag(wxLC_VRULES);
 
     // Check if we need to do anything ourselves: either draw the rules or, in
-    // case of using dark mode under Windows 11, erase the unwanted separator
+    // case of using dark mode under Windows, erase the unwanted separator
```

Comment:
Very minor, but it doesn't make sense to say "under Windows" in this Windows-only code:

```suggestion
    // case of using dark mode, erase the unwanted separator
```